src/Components/Common/Avatar.tsx

```diff
@@ -43,57 +43,63 @@ const initials = (name: string): string => {
 };
 
 interface AvatarProps {
+  id?: string;
   colors?: [string, string];
   name: string;
   imageUrl?: string;
   className?: string;
 }
 
 const Avatar: React.FC<AvatarProps> = ({
+  id,
   colors: propColors,
   name,
   imageUrl,
   className,
 }) => {
   const [bgColor] = propColors || toColor(name);
-  const [width, setWidth] = useState(0);
```

Comment:
@shivankacker please do review these changes; If we can keep it simple it would be great.

Reply:
@bodhish tested out, it is working great. The approach is better and simpler than the previous one.